File: programs/issubsequence.cs
public class Solution {
    public bool IsSubsequence(string s, string t) {
        int l = 0;
        int r = 0;

        Console.WriteLine(s.Length);

        while(l < s.Length && r < t.Length){
            if(s[l] == t[r]){
                l++;
            }

            r++;
        }

        if(l == s.Length){
            return true;
        }

        return false;
    }
}

File: programs in python/is-subsequence.py
class Solution:
    def isSubsequence(self, s: str, t: str) -> bool:
        # if s is subsequence of t then traverse through s and t and if 
        # we can travse through all of s then s is subsequence else not . 

        i = 0 
        j = 0 

        while i < len(s) and j < len(t): 
            if s[i] == t[j]:
                i+= 1
            j += 1

        print(i)
        return i == len(s)  